bodhi/models/models.py

```diff
@@ -1792,6 +1792,12 @@ def days_in_testing(self):
             return 0
 
     @property
+    def date_locked(self):
+        """ Return the date when the update has been locked """
+        if self.locked:
+            return datetime.utcnow()
```

Comment:
Won't this always return ``utcnow()``?
So today, it won't it show in the template as:
``
This update is currently locked since <small>2016-05-03 07:59:51 (UTC)</small> and cannot be modified.
``
In one hour:
``
This update is currently locked since <small>2016-05-03 08:59:51 (UTC)</small> and cannot be modified.
``
And tomorrow:
``
This update is currently locked since <small>2016-05-04 07:59:51 (UTC)</small> and cannot be modified.
``
?